packages/functional-tests/tests/oauth/signinTokenCode.spec.ts

```diff
@@ -65,7 +65,32 @@ test.describe('severity-2 #smoke', () => {
       pages: { page, signin, relier, signinTokenCode },
       testAccountTracker,
     }) => {
-      test.fixme(true, 'TODO in FXA-9931');
+      // The `sync` prefix is needed to force confirmation.
+      const credentials = await testAccountTracker.signUpSync();
+
+      await relier.goto(toQueryString(queryParameters));
+      // Click the Email First flow, which should direct to the sign in page
+      await relier.clickEmailFirst();
+      // Enter email, then enter password
+      await signin.fillOutEmailFirstForm(credentials.email);
+      await signin.fillOutPasswordForm(credentials.password);
+      // Enter invalid code, ensure it doesn't work
+      await expect(page).toHaveURL(/signin_token_code/);
+
+      // Correctly submits the token code and navigates to oauth page
+      const code = await target.emailClient.getVerifyLoginCode(
+        credentials.email
+      );
+      await signinTokenCode.fillOutCodeForm(code);
+
+      await expect(page).toHaveURL(/notes\/fxa/);
+    });
+
+    test('verified - invalid code', async ({
+      target,
```

Comment:
target seems unused in this case?
```suggestion
```

Reply:
Thank you! I've cleaned up the comments and removed that stray `target`.